index.bs

```diff
@@ -994,6 +994,22 @@ Consider adding such features only in cases when the overall user experience is
 A canonical example of this is blocking rendering in order to download and process a stylesheet. 
 The alternative user experience is a flash of unstyled content, which is undesirable.
 
+<h3 id="html-idl-must-by-synced">Keep attributes in sync</h3>
+
+If your feature introduces new attributes,
+try to ensure the attributes are synchronized.
+
+This means any new content attribute
+should always have a corresponding IDL attribute with the same name,
+and the state between the two should be kept synchronized.
+Carving out a synchronized IDL attribute with inconsistent naming
+results in confusion, and should be avoided at all costs.
+
+<div class="note">
+This does not hold the other way around.
+A new IDL attribute does not always warrant a content attribute counterpart.
```

Comment:
This needs an example of an IDL attribute that does not correspond to an HTML attribute.